### PR TITLE
Automate release

### PR DESCRIPTION
**Pull request by robaerd**
## Basics

These points need to be fulfilled for every PR:

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Comment by mpranj** [comment]
Sorry, we left our reviews in the restyled version of the PR #3525.

**Review by mpranj — CHANGES_REQUESTED** [review]
This looking great already!

Please fix the two little comments and add some release notes. If you check the [CI output](https://build.libelektra.org/blue/organizations/jenkins/libelektra/detail/PR-3524/2/pipeline), it complains that there are no release notes, so it does not run all tests.

After that I think we can merge this and test it on the actual Jenkins!

File: scripts/release/release.sh [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+install_elektra() {
+	echo "Installing elektra..."
+	rm -rf $BUILD_DIR
+	mkdir $BUILD_DIR
+	cd $BUILD_DIR
+	cmake -DBUILD_SHARED=ON -DBUILD_FULL=ON -DBUILD_STATIC=ON ..
+	cd ..
+	cmake --build build
+	cd $BUILD_DIR
```
[the remark this answers is not in the document]
Comment (robaerd) [reply scripts/release/release.sh]: I changed the cmake command to make so it is more consistent, thanks.

**Review by mpranj — APPROVED** [review]
Thank you, great work.

I'll merge this as it is important for the release, even though one build is still not finished. That build is not really related to these changes though, so it should be fine. 🙈

**Comment by mpranj** [comment]
@robaerd you should now be able to set up and test the pipeline on Jenkins.

**Pull request merged**